### PR TITLE
fix/Upgrade action to node 20

### PR DESCRIPTION
**Pull request by jsconan**
Try to fix action issue  still using node@12

### CODE REVIEW

**Review by progys — APPROVED** [review]

File: .github/workflows/release-package.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+jobs:
+    auto-release:
+        if: github.event.pull_request.merged == true
+        name: Automated package release
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Clone the repository
+              uses: actions/checkout@v3
```
Comment (taorafael) [inline .github/workflows/release-package.yml]: @rahulfoc1990 is updating the actions/checkout to use the @v4 instead, might want to go with that https://oat-sa.atlassian.net/browse/OPS-534

File: action.yml [hunk 2]
```diff
@@ -9,5 +9,5 @@ outputs:
   version:
     description: 'The version resolved from commits'
 runs:
-  using: 'node12'
+  using: 'node20'
```
Comment (taorafael) [inline action.yml:12]: +1

**Review by taorafael — CHANGES_REQUESTED** [review]
actions/checkout@v3 seems to be deprecated as well:
https://oat-sa.atlassian.net/browse/OPS-534

**Review by taorafael — APPROVED** [review]
lgtm, deprecations addressed, thanks and good job @jsconan

**Pull request merged**